backend/src/chats/chats.controller.ts

```diff
@@ -32,6 +39,11 @@ export class ChatsController {
     return this.chatsService.mapChatsToChatsWithName(chats, user);
   }
 
+  @Get('all')
+  async findAllChats(): Promise<ChatEntity[]> {
+    return await this.chatsService.listAllChats();
+  }
```

Comment:
Acho que seguindo REST `/chats` é mais comum do que `/chats/all` pra pegar todos 
```suggestion
  @Get('/')
  async findAllChats(): Promise<ChatEntity[]> {
    return await this.chatsService.listAllChats();
  }
```

Reply:
Sim, porem ja existe um handler get para essa rota, e esse handler retorna os chats da qual o usuario logado participa, apenas. Essa nova rota retorna todos chats, n apenas os que o usuario faz parte. Por esse motivo criei esse /all

Reply:
Saquei! Talvez depois essa que tu fez a gente torna como padrão e a outra que retorna os chats que o user participa pode ser a chats/mine , chats/joined ou algo assim